app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/RemoveRecordConnector.scala

```diff
@@ -63,15 +63,20 @@ class RemoveRecordConnector @Inject() (
    */
   private def headers(correlationId: String, accessToken: String, forwardedHost: String)(implicit
     hc: HeaderCarrier
-  ): Seq[(String, String)] =
+  ): Seq[(String, String)] = {
+    (HeaderNames.ContentType -> MimeTypes.JSON)
```

Comment:
This may be rewrite:
```suggestion
    val headers = commonHeaders(correlationId, accessToken, forwardedHost)

    val extraHeaders = if (appConfig.isDrop2Enabled) {
        Seq(HeaderNames.ContentType -> MimeTypes.JSON)
    } else if (appConfig.acceptHeaderEnabled) {
        Seq(HeaderNames.ContentType -> MimeTypes.JSON, HeaderNames.ClientId -> getClientId)
    } else
        Seq(
          HeaderNames.Accept      -> MimeTypes.JSON,
          HeaderNames.ContentType -> MimeTypes.JSON,
          HeaderNames.ClientId    -> getClientId
        )

    headers ++ extraHeaders
```